### PR TITLE
Pre 1.0 prep

### PR DESCRIPTION
**Pull request by Cadair**
@DanRyanIrish Review on the release announcement would be great.

I will post it as a blog post on the sunpy website and send it as an email to the sunpy and astropy lists.

### CODE REVIEW

File: RELEASE.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+The SunPy project is very happy to announce the release of a new package "ndcube".
+
+ndcube is a package built to represent a wide variety of data, of any number of
+dimensions, along with coordinate information described by FITS-WCS. ndcube is
+based on the astropy NDData class, and adds functionality for slicing both the
+data and the WCS simultaneously, plotting with matplotlib and support for extra
+coordinate information along any of the axes not described by WCS.
```
Comment (DanRyanIrish) [inline RELEASE.rst]: Since `NDCubeSequence` is mentioned in the next paragraph, `NDCube` should be mentioned in this one.

File: setup.cfg [hunk 2]
```diff
@@ -39,17 +39,12 @@ exclude = extern,sphinx,*parsetab.py
...
 url = http://sunpy.org
 edit_on_github = True
 github_project = sunpy/ndcube
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-version = 0.1.dev
-
-[entry_points]
-
-# astropy-package-template-example = packagename.example_mod:main
+version = 1.0.dev
```
Comment (DanRyanIrish) [inline setup.cfg]: Should version just be `1.0`?  Not `1.0.dev`?

File: setup.cfg [hunk 3]
```diff
@@ -39,17 +39,12 @@ exclude = extern,sphinx,*parsetab.py
 [metadata]
 package_name = ndcube
 description = A base package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
```
Comment (DanRyanIrish) [inline setup.cfg]: Instead of "spatially aware", perhaps we shoudl say "coordinate aware".

File: RELEASE.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
+The SunPy project is very happy to announce the release of a new package "ndcube".
+
+ndcube is a package built to represent a wide variety of data, of any number of
```
Comment (DanRyanIrish) [inline RELEASE.rst]: How about "ndcube is a package built for handling, inspecting and visualizing a wide variety of data, of any number of dimensions, along with coordinate information described by FITS-WCS."

File: .mailmap [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Daniel Ryan <ryand5@tcd.ie> DanRyanIrish <ryand5@tcd.ie>
+Ankit Baruah <ankit.baruah1@gmail.com> abit2 <ankit.baruah1@gmail.com>
+Nabil Freij <nabil.freij@gmail.com> Nabobalis <nabil.freij@gmail.com>
+Ankit Baruah <ankit.baruah1@gmail.com> ankit <ankit.baruah1@gmail.com>
```
[the remark this answers is not in the document]
Comment (Cadair) [reply .mailmap]: This file fixes up the author information in peoples commits. I use it to make the output of `git shortlog -sn` print peoples actual names. If the commits already have your real name in them then you don't need an entry in this file.

File: setup.cfg (hunk 2 above)
Comment (Cadair) [reply setup.cfg]: I will bump to 1.0 later in the release process.

**Comment by DanRyanIrish** [comment]
lgtm pending Travis and Appveyor passing.

**Pull request merged**